### PR TITLE
News  and event sample page created

### PR DESCRIPTION
**Pull request by AmilaIndika789**
News and event page sample is created. Added CSS as <style jsx>  Any suggestions ?

### CODE REVIEW

**Comment by shyaman** [comment]
I've already made changes these changes on 'news'  page. @AmilaIndika789 you can check them out on the **[feature/news-and-events](https://github.com/shyaman/Sport-events-and-players-management-platform)** branch. But I think some of your changes are missing there.

- The footer
- Modifications to the event slots

And I have made the JSX as global in the [header.js](https://github.com/shyaman/Sport-events-and-players-management-platform/blob/feature/news-and-events/components/Header.js).

File: pages/news.js [hunk 1]
```diff
@@ -1,7 +1,253 @@
-import Layout from '../components/Layout.js'
+import Layout from '../components/Layout.js';
+
+const img = {
```
Comment (shyaman) [inline pages/news.js]: Do we need this const?

Comment (AmilaIndika789) [reply pages/news.js]:
```suggestion
const img = {
```
@shyaman No it is not necessary. remove that.

**Review by shyaman — APPROVED** [review]
All seems fine.

**Pull request merged**